src/compile/data/parse.ts

```diff
@@ -125,6 +119,12 @@ export function parseData(model: Model): DataComponent {
     head = last;
   }
 
+  const parse = ParseNode.make(model);
```

Comment:
See main PR comment. The issue is that the field may come from the secondary source of a lookup.